AndroidManifest.xml

```diff
@@ -154,7 +161,6 @@
         <intent-filter>
             <data android:scheme="mailto"/>
```

Comment:
mailto is probably not needed anymore here?

Reply:
If I remove only this line, then Android Studio is mad at me as there is a `data` element missing. If I remove the whole block then I can't open mailto links with DC at all anymore. And if I then add `mailto` and `action.VIEW` up to ShareActivity, then mailto links can be shared to DC but DC doesn't find the content.

So, I'd actually propose to merge this PR as is because everything is working fine (at least sharing from NewPipe, SuperFreezZ and all the mailto links in the testing document I just created). Maybe code style is not the best as we still have multiple activities caring about sharing but it's the minimal-change solution, hopefully with the lowest risk of breaking something.

Reply:
hm, that's weird. i'll have look at that later.